### ISSUE TITLE
Is there a .well-known folder?

### ISSUE THREAD

**Issue by IraklisElef** [emilbaehr/automatic-app-landing-page#77]
Hello,

I am trying to add my domain to the `Associated Domains` in Xcode. In order to do that, I need to place a file named `apple-app-site-association` inside the `.well-known` folder. How can I do that with this package? Thank you in advance!

Sincerely,
Iraklis Eleftheriadis

**Comment by visamplify** [comment]
@IraklisElef : Did you find a solution for this?

**Comment by IraklisElef** [comment]
> @IraklisElef : Did you find a solution for this?

Yes I did. I simple had to create a `.well-known` folder and inside there create a JSON file named `apple-app-site-association`. Finally, you should format the JSON file like this: 

`{`
  `"webcredentials": {`
   ` "apps": [`
     ` "<Apple Developer Team ID>.<App Package Name>"`
   ` ]`
 ` }`
`}`

For example:

`{`
  `"webcredentials": {`
   ` "apps": [`
     ` "8W66MEA7DD.com.spotify.client"`
   ` ]`
 ` }`
`}`
